src/render/js/components/UuidItem.js

```diff
@@ -0,0 +1,41 @@
+import m from 'mithril'
+import { clipboard } from 'electron'
+
+import { deleteUuid } from '../actions'
+
+const copyItem    = (clip, uuid) => () => clip.writeText(uuid)
+const deleteItem  = (dispatch, action, index) => () => dispatch(action(index))
+
+function controller(attrs) {
+  const { uuid, dispatch, index } = attrs
+
+  const copy    = copyItem(clipboard, uuid)
+  const remove  = deleteItem(dispatch, deleteUuid, index)
```

Comment:
`deleteUuid` in the controller is not PURE...need to change `deleteItem`
```javascript
const actionDispatch = action => (dispatch, index) => () => dispatch(action(index))
const deleteItem = actionDispatch(deleteUuid)
```
Then remove can become
```javascript
const remove = deleteItem(dispatch, index)
```